src/java.desktop/macosx/native/libawt_lwawt/awt/JavaAccessibilityUtilities.m

```diff
@@ -51,6 +51,7 @@
 
 NSSize getAxComponentSize(JNIEnv *env, jobject axComponent, jobject component)
 {
+    GET_CACCESSIBILITY_CLASS_RETURN(nil);
```

Comment:
Please, use `NSZeroSize` as the return value here.

Reply:
Done